NickvisionMoney.GNOME/Blueprints/account_view.blp

```diff
@@ -175,16 +175,27 @@ Adw.Bin _root {
         
         Adw.PreferencesGroup {
           title: _("Calendar");
-          header-suffix: Gtk.Button _resetCalendarFilterButton {
-            tooltip-text: _("Reset To Today");
+          header-suffix: Gtk.Box 
+          {
+            orientation: horizontal;
 
-            Adw.ButtonContent {
-              icon-name: "today-symbolic";
-              label: _("Today");
+            Gtk.Button _selectWholeMonthButton {
+              tooltip-text: _("Select current month");
```

Comment:
```suggestion
              tooltip-text: _("Select Current Month");
```

To match other tooltip labels.

Reply:
I added everything and made sure it built before committing!! Should be good to go